### DISCUSSION TITLE
how the hard-break-extension listen the 'Enter' event，now it only supports Mod-Enter and Shift-Enter

### DISCUSSION THREAD

**Discussion by Just-focus** [remirror/remirror#1236] [Q&A]
how the hard-break-extension listen the 'Enter' event，now it only support Mod-Enter and Shift-Enter

![image](https://user-images.githubusercontent.com/18374969/133207404-8007d4e5-c436-413a-9efd-36e0ba7f1296.png)

![image](https://user-images.githubusercontent.com/18374969/133207432-16491484-d531-4caf-960a-a259f088b50d.png)

**Comment by ocavue** [comment]
`HardBreakExtension` doesn't bind to the `Enter` key, `KeymapExtension` does. By default, `Enter` and `Mod-Enter` has different behaviors. 

https://github.com/ProseMirror/prosemirror-commands/blob/ab3dfd169f8710fd6c4b37209c0c769a9e127ce0/src/commands.js#L608
